Project2/package.json

```diff
@@ -10,12 +10,13 @@
   },
   "dependencies": {
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
```

Comment:
I think if you remove the double "react-dom" then the rest will be fine. Put the pull request in so you can finish and merge it. -Travis